doc/api/fs.md

```diff
@@ -1524,11 +1524,27 @@ If `options` is a string, then it specifies the encoding. Example:
 ```js
 fs.readFile('/etc/passwd', 'utf8', callback);
 ```
+*Note*: When the path is a directory, the behavior of
+`fs.readFile()` and [`fs.readFileSync()`][] is platform-specific. On macOS, Linux
+and Windows, an error will be returned. On FreeBSD, the contents of the files
```

Comment:
@Trott is not just being picky. It's called the "oxford comma" and we ask you its use in our [style guide](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md)